lib/manager/nuget/extract.ts

```diff
@@ -1,18 +1,22 @@
 import { logger } from '../../logger';
 import { get } from '../../versioning';
-import { PackageDependency, ExtractConfig, PackageFile } from '../common';
+import {
+  PackageDependency,
+  PackageFile,
+  ExtractPackageFileConfig,
+} from '../common';
 
-export function extractPackageFile(
-  content: string,
-  packageFile: string,
-  config: ExtractConfig = {}
-): PackageFile {
-  logger.trace(`nuget.extractPackageFile(${packageFile})`);
+export function extractPackageFile({
+  fileContent,
+  fileName,
+  config = {},
```

Comment:
```suggestion
  config,
```

Reply:
This would require a change to https://github.com/souravdasslg/renovate/blob/33ebe3435d807c75366694cddeb2a37c7ab69128/lib/manager/nuget/extract.ts#L15 too